.github/scripts/check-orphaned-issues.sh

```diff
@@ -0,0 +1,139 @@
+ACTIVE_PROJECT_COUNT=$((ACTIVE_PROJECT_COUNT + 1))
+ARCHIVED_PROJECT_COUNT=$(echo "$ARCHIVED_PROJECTS" | grep -o "," | wc -l | tr -d ' ')
+ARCHIVED_PROJECT_COUNT=$((ARCHIVED_PROJECT_COUNT + 1))
+
+if [[ -z "$ACTIVE_PROJECTS" ]]; then
+  ACTIVE_PROJECT_COUNT=0
+fi
+
+if [[ -z "$ARCHIVED_PROJECTS" ]]; then
+  ARCHIVED_PROJECT_COUNT=0
+fi
+
+echo "  Found $ACTIVE_PROJECT_COUNT active project(s)"
+echo "  Found $ARCHIVED_PROJECT_COUNT archived project(s)"
+
+# Step 2: Build blended search query for orphaned issues
+# We want issues that are:
+# - Open
+# - Either have no project assignment OR only assigned to archived projects (not to any active project)
+
+# Part 1: Issues with no project assignment
+SEARCH_QUERY_NO_PROJECT="repo:${ORG_NAME}/${REPO_NAME} is:issue is:open no:project"
+
+# Part 2: If we have archived projects, build a query for issues assigned to archived projects
+# but NOT assigned to any active projects
+SEARCH_QUERY_ARCHIVED=""
+if [[ -n "$ARCHIVED_PROJECTS" && -n "$ACTIVE_PROJECTS" ]]; then
+  IFS=',' read -ra ARCHIVED_ARRAY <<< "$ARCHIVED_PROJECTS"
+  IFS=',' read -ra ACTIVE_ARRAY <<< "$ACTIVE_PROJECTS"
+  
+  # Build OR query for archived projects
+  ARCHIVED_OR_PARTS=()
+  for archived_proj in "${ARCHIVED_ARRAY[@]}"; do
+    ARCHIVED_OR_PARTS+=("project:${ORG_NAME}/${archived_proj}")
+  done
+  
+  # Combine with OR and wrap in parentheses
+  ARCHIVED_OR_QUERY=$(IFS=' OR '; echo "${ARCHIVED_OR_PARTS[*]}")
+  
+  # Add negation for all active projects
+  ACTIVE_NEGATIONS=""
+  for active_proj in "${ACTIVE_ARRAY[@]}"; do
+    ACTIVE_NEGATIONS="${ACTIVE_NEGATIONS} -project:${ORG_NAME}/${active_proj}"
+  done
+  
+  # Complete query for archived project issues
+  SEARCH_QUERY_ARCHIVED="repo:${ORG_NAME}/${REPO_NAME} is:issue is:open (${ARCHIVED_OR_QUERY})${ACTIVE_NEGATIONS}"
+fi
+
+# Step 3: Count orphaned issues using both queries
+echo "Counting orphaned issues..."
+TOTAL_ORPHANED=0
+
+# Count issues with no project
+RESULT=$(gh api graphql -f query="
+  query(\$searchQuery: String!) {
+    search(query: \$searchQuery, type: ISSUE, first: 1) {
+      issueCount
+    }
+  }" -F searchQuery="$SEARCH_QUERY_NO_PROJECT")
+
+COUNT_NO_PROJECT=$(echo "$RESULT" | jq -r '.data.search.issueCount')
+TOTAL_ORPHANED=$COUNT_NO_PROJECT
+echo "  Issues with no project: $COUNT_NO_PROJECT"
+
+# Count issues in archived projects (if query exists)
+if [[ -n "$SEARCH_QUERY_ARCHIVED" ]]; then
+  RESULT=$(gh api graphql -f query="
+    query(\$searchQuery: String!) {
+      search(query: \$searchQuery, type: ISSUE, first: 1) {
+        issueCount
+      }
+    }" -F searchQuery="$SEARCH_QUERY_ARCHIVED")
+  
+  COUNT_ARCHIVED=$(echo "$RESULT" | jq -r '.data.search.issueCount')
+  TOTAL_ORPHANED=$((TOTAL_ORPHANED + COUNT_ARCHIVED))
+  echo "  Issues only in archived projects: $COUNT_ARCHIVED"
+fi
+
+echo "  Total orphaned: $TOTAL_ORPHANED"
+
+# Step 4: Generate GitHub search URL
```

Comment:
given that we can only generate the URL for the no project issues, let's change this to do things slightly differently for each use case:
- For the issues with no projects, we should include the count in the Slack message (if there are any), as well as the Github URL link to find those issues
- For the issues that are only assigned to inactive projects, we should include the name of the issue, which is linked to the issue itself

This way, all potentially orphaned issues will be navigable from Slack and the appropriate person can review their status